extras/ams_wrapper/start_ams.sh

```diff
@@ -51,5 +56,5 @@ esac
 done
 
 . /ie-serving-py/.venv/bin/activate
-/ie-serving-py/start_server.sh ie_serving model --model_path /opt/models/vehicle_detection_adas --model_name vehicle_detection_adas --port $OVMS_PORT & 
-python /ams_wrapper/src/wrapper.py --port $AMS_PORT
+/ie-serving-py/start_server.sh ie_serving model --model_path /opt/models/vehicle_detection_adas --grpc_workers 10 --nireq 10 --model_name vehicle_detection_adas --port $OVMS_PORT &
```

Comment:
We would probably want to parameterize  OVMS `grpc_workers` and `nireq` too.
It will change though, when we will enable multiple models in AMS.